archinstall/user_configuration.json

```diff
@@ -0,0 +1,46 @@
+{
+    "additional-repositories": [
+        "multilib"
+    ],
+    "audio": "pulseaudio",
+    "bootloader": "grub-install",
+    "config_version": "2.5.1",
+    "debug": false,
+    "harddrives": [
+        "/dev/sda"
+    ],
+    "keyboard-layout": "de_CH-latin1",
+    "mirror-region": {
+        "Switzerland": {
+            "http://mirror.init7.net/archlinux/$repo/os/$arch": true,
+            "http://mirror.puzzle.ch/archlinux/$repo/os/$arch": true,
+            "http://pkg.adfinis.com/archlinux/$repo/os/$arch": true,
+            "https://mirror.init7.net/archlinux/$repo/os/$arch": true,
+            "https://mirror.puzzle.ch/archlinux/$repo/os/$arch": true,
+            "https://mirror.ungleich.ch/mirror/packages/archlinux/$repo/os/$arch": true,
+            "https://pkg.adfinis.com/archlinux/$repo/os/$arch": true,
+            "https://theswissbay.ch/archlinux/$repo/os/$arch": true
+        }
+    },
+    "nic": {
+        "dhcp": true,
+        "dns": null,
+        "gateway": null,
+        "iface": null,
+        "ip": null,
+        "type": "iso"
+    },
+    "no_pkg_lookups": false,
+    "offline": false,
+    "packages": [
+        "ansible",
+        "git",
+        "openssh"
+    ],
+    "profile": {
+        "path": "/usr/lib/python3.10/site-packages/archinstall/profiles/minimal.py"
+    },
+    "script": "guided",
+    "silent": false,
+    "version": "2.5.1"
+}
```

Comment:
I'd remove the archinstall configs. First of all the ansible automation tries to configure `syslinux` as bootloader however the one set in the confguration is `grub`. Second the partitioning scheme does not match the one in the install guide which is part of this repository. This could lead to confusion. :-)